### DISCUSSION TITLE
search results

### DISCUSSION THREAD

**Discussion by roomone** [gollum/gollum#1993] [General]
Hi All, I am a newbie to gollum, I really like it and i think it will work well for my needs. I have a question about the search results. when the search returns a result i can only select the file name and it brings top of the file, is there a way to set this up so it brings me to the location within the file?

![Screenshot from 2023-08-03 19-29-21](https://github.com/gollum/gollum/assets/61564169/9b6ed0f8-8faf-494a-9625-3dcfb7fdef5a)

in the example above can i go to the location of "variables" within the  cpp_tutorial.adoc file?

**Comment by bartkamphorst** [comment]
Unfortunately this functionality does not exist at the moment.

**Reply by roomone** [reply]
thanks for the  feedback, are there any plans to include it in the future?

**Reply by bartkamphorst** [reply]
There's no concrete plans for this at the moment. Of course, gollum is an open source project, so if you or anyone else would be willing to implement this, I'd be happy to support the efforts.

**Reply by dometto** [reply]
Not currently but I think it's a good suggestion, and it needn't be that difficult to implement. It would be great if we could gather suggestions and ideas about how the search could be better (within feasible limits -- something like [elastic search integration](#1768) would be very hip but it's a large project) and think about what would need to change on the backend to make this possible. It's very useful to hear from users about what they're missing, so thanks for letting us know and don't hesitate to say more about what could be make the search function better!

**Reply by roomone** [reply]
thanks for the feedback and all your hard work
